lib/client.rb

```diff
@@ -0,0 +1,81 @@
+class Client
+  attr_accessor :id
+
+  include Redis::Objects
+  EXPIRATION_INTERVAL = REDIS_CONFIG.fetch('client_session_timeout', 300)
```

Comment:
Do we need a `.to_i` here, or is the YAML parser smart enough to do that for us?